app/models/message.rb

```diff
@@ -0,0 +1,4 @@
+class Message < ApplicationRecord
```

Comment:
Is it worth making the `role` and `status` an enum? Rails supports string enum values now.